airbyte_cdk/sources/declarative/decoders/composite_decoder.py

```diff
@@ -0,0 +1,98 @@
+import gzip
+import json
+import logging
+from abc import ABC, abstractmethod
+from dataclasses import dataclass
+from io import BufferedIOBase
+from typing import Any, Generator, MutableMapping, Optional
+
+import pandas as pd
+import requests
+from numpy import nan
+
+from airbyte_cdk.sources.declarative.decoders.decoder import Decoder
+
+logger = logging.getLogger("airbyte")
+
+
+@dataclass
+class Parser(ABC):
+    @abstractmethod
+    def parse(
+        self, data: BufferedIOBase, *args, **kwargs
```

Comment:
Just to make sure I understand: Why do we have `*args, **kwargs` in the interface?

Reply:
no reason (or real use-case) for now, can be deleted.